### PR TITLE
Adding SourceLink support to DurableTask.Core project

### PR DESCRIPTION
**Pull request by gled4er**
Hello @cgillum,

May I ask you to review this small PR for adding SourseLink support to `DurableTask.Core` project?

Thank you!

### CODE REVIEW

**Comment by gled4er** [comment]
Hello @simonporter,

I moved the SourceLink configuration to `DurableTask.props`.

May I ask you to take a look and let me know what should be fixed next?

Thank you!

File: tools/DurableTask.props [hunk 1]
```diff
@@ -13,7 +13,18 @@
     <RunCodeAnalysis Condition=" '$(TargetFramework)' == 'netstandard2.0'">False</RunCodeAnalysis>
     <CodeAnalysisTreatWarningsAsErrors>True</CodeAnalysisTreatWarningsAsErrors>
     <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
+    <!-- SourseLink Settings-->
```
Comment (simonporter) [inline tools/DurableTask.props]: Nit: typo in Source

File: src/DurableTask.Core/DurableTask.Core.csproj [hunk 2]
```diff
@@ -5,7 +5,16 @@
     <TargetFrameworks>netstandard2.0;net451</TargetFrameworks>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <PackageId>Microsoft.Azure.DurableTask.Core</PackageId>
-  </PropertyGroup>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
```
[the remark this answers is not in the document]
Comment (gled4er) [reply src/DurableTask.Core/DurableTask.Core.csproj]: Thank you @simonporter! 

I will move the configuration.

File: tools/DurableTask.props (hunk 1 above)
Comment (gled4er) [reply tools/DurableTask.props]: Thank you for catching this!

I will fix it.

**Comment by gled4er** [comment]
Hello @simonporter,

I implemented the required changes.


Thank you for your support on this PR!

**Pull request merged**